books.js

```diff
@@ -0,0 +1,78 @@
+const addBtn = document.getElementById('add-btn');
+const bookList = document.querySelector('#book-list');
+
+let books = [];
+
+// Displaying Added Element
+const displayBooks = (id, title, author) => {
+  const li = document.createElement('li');
+  const br = document.createElement('br');
+  li.innerHTML = `
+  <h2>${title}</h2>
+  <h2>${author}</h2>
+  <hr>`;
+  const removeBookBtn = document.createElement('button');
+  removeBookBtn.textContent = 'Remove';
+  li.insertBefore(removeBookBtn, li.lastElementChild);
+  li.appendChild(br);
+  bookList.appendChild(li);
+  removeBookBtn.addEventListener('click', () => {
+    books = books.filter((book) => {
+      if (book.id !== id) {
+        return true;
+      }
+      return books;
+    });
+    localStorage.setItem('books', JSON.stringify(books));
+    li.remove();
+  });
+};
```

Comment:
- [ ] Great job on adding new book 👏 . but you have a problem with updating your local storage after removing a book because your local storage does not update with the new information and shows the same book every time you reload the page, Kindly fix the issue